src/lib/discord/commands/admin.ts

```diff
@@ -0,0 +1,215 @@
+import { SlashCommandBuilder } from '@discordjs/builders'
+import { faker } from '@faker-js/faker'
+import { EmbedBuilder } from 'discord.js'
+import { getUserAccess } from '$discord/get-user-access'
+import { prisma } from '$lib/db'
+import {
+  postDiscussion,
+  postAnswer,
+  markAnswered,
+  lockDiscussion,
+} from '$lib/github/queries'
+import { repositoriesWithDiscussions as repositories } from './_repositories'
+import { isThreadWithinHelpChannel } from '../support'
+import type {
+  GuildMember,
+  Message,
+  ChatInputCommandInteraction,
+  InteractionReplyOptions,
+  ThreadChannel,
+} from 'discord.js'
+
+const userIdToUsername = new Map<string, user>()
+
+// const iconMap = new Map<string, string>([
+//   ['Admin', "https://github.com/esauerbo1/Images/blob/main/Admin.svg"],
+//   ['Bot', 'https://github.com/esauerbo1/Images/blob/main/Bot.svg'],
+//   ['Amplify Bot Dev', 'https://github.com/esauerbo1/Images/blob/main/Amplify-Bot-Dev.svg'],
+//   ['Moderator', 'https://github.com/esauerbo1/Images/blob/main/Moderator.svg'],
+//   ['Amplify Staff', 'https://github.com/esauerbo1/Images/blob/main/Amplify-Staff.svg'],
+//   ['AWS Staff', 'https://github.com/esauerbo1/Images/blob/main/AWS-Staff.svg'],
+//   ['Community Builder', 'https://github.com/esauerbo1/Images/blob/main/Community-Builder.svg'],
+//   ['Contributor', 'https://github.com/esauerbo1/Images/blob/main/Contributor.svg'],
+//   ['Meetup Organizer', 'https://github.com/esauerbo1/Images/blob/main/Meetup-Organizer.svg'],
+//   ['Amplify Guru', 'https://github.com/esauerbo1/Images/blob/main/Amplify-Guru.svg'],
+//   ['@everyone', 'https://github.com/esauerbo1/Images/blob/main/%40everyone.svg'],
+// ])
+
+type user = {
+  username: string
+  avatar: string
+  highestRole: string
+}
+
+function getUser(userId: string, guildMember: GuildMember | null) {
+  if (!userIdToUsername.get(userId)) {
+    let role = '@everyone'
```

Comment:
```suggestion
    let role = 'at-everyone'
```

minor suggestion to avoid creating a mention